src/gordon_gcp/plugins/service/__init__.py

```diff
@@ -16,17 +16,18 @@
 
 from gordon_gcp.plugins.service import enricher
 from gordon_gcp.plugins.service import event_consumer
+from gordon_gcp.plugins.service import gdns_publisher
 # Mainly for easier documentation reading
-from gordon_gcp.plugins.service.enricher import *  # noqa: F403
-from gordon_gcp.plugins.service.event_consumer import *  # noqa: F403
-from gordon_gcp.plugins.service.gdns_publisher import *  # noqa: F403
+from gordon_gcp.plugins.service.enricher import *  # noqa: F401,F403
```

Comment:
I have no idea why once I added L19 that these started to fail (unused import blahblahblah). freakin' flake8 ಠ_ಠ.